src/clj/rems/css/style_utils.clj

```diff
@@ -0,0 +1,79 @@
+(ns rems.css.style-utils
+  "CSS stylesheets are generated by garden automatically when
+  accessing the application on a browser. The garden styles can also
+  be manually compiled by calling the function `rems.css.styles/screen-css`.
+
+  For development purposes with Figwheel the styles are rendered to
+  `target/resources/public/css/:language/screen.css` whenever this ns is evaluated
+  so that Figwheel can autoreload them."
+  (:require [clojure.string :as str]
+            [clojure.test :refer [deftest is testing]]
+            [clojure.tools.logging :as log]
+            [compojure.core :refer [GET defroutes]]
+            [garden.color :as c]
+            [garden.core :as g]
+            [garden.def :refer [defkeyframes]]
+            [garden.selectors :as s]
+            [garden.stylesheet :as stylesheet]
+            [garden.units :as u]
+            [medley.core :refer [map-vals remove-vals]]
+            [mount.core :as mount]
+            [rems.config :refer [env]]
+            [rems.context :as context]
+            [ring.util.response :as response]))
+
+
+;; Customazable theme related functions
+
+(defn get-theme-attribute
+  "Fetch the attribute value from the current theme with fallbacks.
+
+  Keywords denote attribute lookups while strings are interpreted as fallback constant value."
+  [& attr-names]
+  (when (seq attr-names)
+    (let [attr-name (first attr-names)
+          attr-value (if (keyword? attr-name)
+                       (get (:theme env) attr-name)
+                       attr-name)]
+      (or attr-value (recur (rest attr-names))))))
+
+(defn resolve-image [path]
+  (when path
+    (let [url (if (str/starts-with? path "http")
+                path
+                (str (get-theme-attribute :img-path "../../img/") path))]
+      (str "url(\"" url "\")"))))
+
+(defn get-logo-image [lang]
+  (resolve-image (get-theme-attribute (keyword (str "logo-name-" (name lang))) :logo-name)))
+
+(defn get-logo-name-sm [lang]
+  (resolve-image (get-theme-attribute (keyword (str "logo-name-" (name lang) "-sm")) :logo-name-sm)))
+
+(defn get-navbar-logo [lang]
+  (resolve-image (get-theme-attribute (keyword (str "navbar-logo-name-" (name lang))) :navbar-logo-name)))
+
+
+;; Fonts of the app 
+
+(defn generate-at-font-faces []
```

Comment:
This thing kind of feels like a style, and not like a style util, so I kinda feel it belongs in rems.css.styles. It's not a big deal so it can stay here too.

Reply:
This is not a style util IMHO too.

Reply:
I also thought about it, actually, but util is a rather fluid term, especially in CSS. the way I see it, it is only used once per codebase as we have just concluded on flowdock, we came to a conclusion that typogrphy is configurable to an extend, so it actually should be a util. I hope I was able to articulate myself clearly.

Reply:
No I do not agree that we reached a conclusion that this definition should be in the utils file.

Reply:
![Uploading photo_2021-03-04 11.39.45.jpeg…]()

Reply:
![photo_2021-03-04 11 39 45](https://user-images.githubusercontent.com/7344718/109945149-a142e880-7cdf-11eb-88a2-e7834789da2d.jpeg)

Reply:
So what should I do then @opqdonut @Macroz ? I think me and @opqdonut are now in agreement but @Macroz does not agree?

Reply:
I would expect the issue texts updated on the board or a summary of your meeting, I guess. I was specifically reacting to this statement that in the previous chat we would have all agreed to put the font definitions into the util file. I think that is misusing the term util and it's just not the right thing to do.

Reply:
Summary of my meeting?

Reply:
You had a meeting with @opqdonut about the refactorings to the styles of which I don't know the results yet. I'm not sure if your agreement refers to that or our chat where I think we did not reach a conclusion about the fonts. I would expect the issues to be updated based on these.

Reply:
We didn't talk about this exact issue (where to put the font-face definitions). I think it's fine here, especially since it isn't an active part of styling, more like styling infra.